### PR TITLE
Video Samples v1

### PR DESCRIPTION
**Pull request by remi**
Update Video samples for v1
 
 - Bump google-cloud-video_intelligence to 0.25.0
 - Update Quickstart   
 - Update Label Detection sample
 - Update Label Detection with local file sample 
 - Updated Shot Change sample
 - Removed Face Detection sample
 - Removed Safe Search sample
 - Added Explicit Content Detection sample
 - Updated specs and README

### CODE REVIEW

**Comment by frankyn** [comment]
Test "can analyze labels from a local file" hangs.

File: video/video_samples.rb [hunk 1]
```diff
@@ -55,12 +55,11 @@ def analyze_labels_local path:
...
 
-  require "base64"
   require "google/cloud/video_intelligence"
 
-  video         = Google::Cloud::VideoIntelligence.new
-  video_file    = File.read path
-  encoded_video = Base64.encode64 video_file
+  video = Google::Cloud::VideoIntelligence.new
+
+  encoded_video = File.binread path
```
Comment (frankyn) [inline video/video_samples.rb]: The variable name should be updated as it's no longer encoded into base64. What about `video_contents`?

**Comment by remi** [comment]
The Video specs pass!

File: video/quickstart.rb [hunk 2]
```diff
@@ -13,39 +13,38 @@
...
+require "google/cloud/video_intelligence"
 
-video_client = Google::Cloud::VideoIntelligence::V1beta1::VideoIntelligenceServiceClient.new
+video_client = Google::Cloud::VideoIntelligence.new
 features     = [:LABEL_DETECTION]
 path         = "gs://demomaker/cat.mp4"
 
 # Register a callback during the method call
-operation = video_client.annotate_video path, features do |operation|
+operation = video_client.annotate_video input_uri: path, features: features do |operation|
```
Comment (frankyn) [inline video/quickstart.rb]: I saw you inlined the value for features for other samples, but not this one. Is it because the line is long?

Comment (remi) [reply video/quickstart.rb]: Would you like this changed? Please let me know either way as soon as you have a moment

**Pull request merged**